app/controllers/users_controller.rb

```diff
@@ -28,9 +28,12 @@ def initialize
   def index
     @new_user = User.find_by_id(params[:user])
 
-    @users = User.with_recent_orders(current_facility)
-                  .order(:last_name, :first_name)
-                  .paginate(:page => params[:page])
+    @users =
+      if all_facility?
+        User.scoped
```

Comment:
Would it be better to have the `with_recent_orders` scope allow the AllFacility, and not apply the `for_facility` scope?